### PR TITLE
add code to run docker statistics

### PR DESCRIPTION
**Pull request by JimHofman**
Added docker statistics to our output.

### CODE REVIEW

**Comment by collinss-jpl** [comment]
Hi @JimHofman, I expected to see an update to the Dockerfile that adds the curl command which downloads the pre-compiled docker-stats-on-exit-shim script into the container. Could you please add a commit with the update, otherwise I don't think this branch will work correctly.

**Comment by JimHofman** [comment]
Sorry Scott, I recommitted and everything is now in place.

**Comment by collinss-jpl** [comment]
Hi @JimHofman, I don't actually see any new commits or changes on this PR, do the recommits still need to be pushed?

**Comment by JimHofman** [comment]
Should be there.

File: .ci/docker/Dockerfile_dswx_hls [hunk 1]
```diff
@@ -14,7 +14,9 @@
 # Author: Scott Collins
 
 # TODO: could this become an argument?
-FROM artifactory-fn.jpl.nasa.gov:16001/gov/nasa/jpl/opera/adt/opera/dswx_hls:interface
+# FROM artifactory-fn.jpl.nasa.gov:16001/gov/nasa/jpl/opera/adt/opera/dswx_hls:interface
+# TODO: Temporary work around while access issues and workarounds are being figured out.
+FROM artifactory-fn.jpl.nasa.gov:16001/gov/nasa/jpl/opera/sds/adt/dswx_hls:interface
```
Comment (riverma) [inline .ci/docker/Dockerfile_dswx_hls:19]: @collins-jpl @JimHofman - what are your thoughts about pushing this binary out to GitHub Packages such that this code is testable in open source?

File: .ci/docker/Dockerfile_dswx_hls [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021, by the California Institute of Technology.
+# Copyright 2022, by the California Institute of Technology.
```
Comment (riverma) [inline .ci/docker/Dockerfile_dswx_hls:1]: Generally speaking, I advise _against_ including the licensing, copyright, and authorship in every code file. It becomes cumbersome to maintain and also (especially for authorship) doesn't quite reflect all contributors to the code. 

My suggestion is to keep the LICENSE file in the repo up-to-date, and leave the authorship to Git commits (and Git blames)

Comment (collinss-jpl) [reply .ci/docker/Dockerfile_dswx_hls:1]: I was under the impression that, despite being an open source project, we were still responsible for including this information for all our production code. But I can't remember if it was in our requirements, or a JPL/NASA mandate though. Could use some clarification on this.

**Comment by collinss-jpl** [comment]
Closing this PR, as we've determined that the docker-stats-on-exit-shim utility is not out-of-the-box compatible with the ubuntu images we base our PGE/SAS containers on. We will be investigating use of the `docker stats` utility to provide similar functionality going forward.

File: .ci/docker/Dockerfile_dswx_hls (hunk 2 above)
Comment (riverma) [reply .ci/docker/Dockerfile_dswx_hls:1]: @collinss-jpl - JPL's stance is to place copyrights in [a central license file](https://github.com/nasa/opera-sds-pge/blob/main/LICENSE.txt#L190) within OSS repos to make maintainability easier for developers. Not a huge deal and no one is going to ping you, but it'll probably save you having to change a code file for a non-code related legalize reason. I'll send you some more info privately.

**Pull request closed without merging**